makejdk.sh

```diff
@@ -141,67 +142,65 @@ setDefaultIfBranchIsNotProvided()
 setWorkingDirectoryIfProvided()
 {
   if [ -z "${WORKING_DIR}" ] ; then
-    echo "${info}WORKING_DIR is undefined so setting to $PWD${normal}"
+    echo "${info}WORKING_DIR is undefined so setting to $PWD${normal}."
     WORKING_DIR=$PWD
   else
-    echo "${info}Working dir is $WORKING_DIR${normal}"
+    echo "${info}Working dir is $WORKING_DIR${normal}."
   fi
 }
 
 setTargetDirectoryIfProvided()
 {
-  echo $info
+  echo "${info}"
   if [ -z "${TARGET_DIR}" ] ; then
     echo "${info}TARGET_DIR is undefined so setting to $PWD"
     TARGET_DIR=$PWD
     # Only makes a difference if we're in Docker
-    echo "If you're using Docker The build artifact will not be copied to the host"
+    echo "If you're using Docker the build artifact will not be copied to the host."
   else
     echo "${info}Target directory is $TARGET_DIR${normal}"
     COPY_TO_HOST=true
-    echo "If you're using Docker we'll copy the build artifact to the host"
+    echo "If you're using Docker we'll copy the build artifact to the host."
   fi
 }
 
 cloneOpenJDKGitRepo()
 {
-  echo $git
+  echo "${git}"
   if [ -d "${WORKING_DIR}/${OPENJDK_REPO_NAME}/.git" ] && [ "$REPOSITORY" == "AdoptOpenJDK/openjdk-jdk8u" ] ; then
     # It does exist and it's a repo other than the AdoptOpenJDK one
-    cd $WORKING_DIR/$OPENJDK_REPO_NAME
+    cd "${WORKING_DIR}/${OPENJDK_REPO_NAME}" || return
     echo "${info}Will reset the repository at $PWD in 10 seconds...${git}"
     sleep 10
     echo "${git}Pulling latest changes from git repo"
     git fetch --all
     git reset --hard origin/$BRANCH
-    echo $normal
-    cd $WORKING_DIR
+    echo "${normal}"
+    cd "${WORKING_DIR}" || return
```

Comment:
>We had a discussion on one of the PR that "${}" is the same as ${}, i noted it for our guideline (see #54)

I think the conclusion there was that `${VAR}` is the same as `$VAR`, quotes are a different thing.

So it could be `"${VAR}"` or `"$VAR"`, but I guess that's a question for #54